tests/misc_test.go

```diff
@@ -77,6 +84,68 @@ var _ = Describe("Observed generation", func() {
 	})
 })
 
+var _ = Describe("SSPOperatorReconcilingProperly metric", func() {
+	var (
+		deploymentRes testResource
+		deployment    = &apps.Deployment{}
+		finalizerName = "ssp.kubernetes.io/temp-protection"
+	)
+
+	BeforeEach(func() {
+		strategy.SkipSspUpdateTestsIfNeeded()
+	})
+
+	AfterEach(func() {
+		Expect(apiClient.Get(ctx, deploymentRes.GetKey(), deployment)).ToNot(HaveOccurred())
+		// remove the finalizer so everything can go back to normal
+		controllerutil.RemoveFinalizer(deployment, finalizerName)
+		Eventually(func() error {
+			return apiClient.Update(ctx, deployment)
+		}, shortTimeout, time.Second).ShouldNot(HaveOccurred())
+		strategy.RevertToOriginalSspCr()
+		waitUntilDeployed()
+	})
+
+	BeforeEach(func() {
+		expectedLabels := expectedLabelsFor("template-validator", common.AppComponentTemplating)
+		deploymentRes = testResource{
+			Name:           validator.DeploymentName,
+			Namespace:      strategy.GetNamespace(),
+			Resource:       &apps.Deployment{},
+			ExpectedLabels: expectedLabels,
+			UpdateFunc: func(deployment *apps.Deployment) {
+				deployment.Spec.Replicas = pointer.Int32Ptr(0)
+			},
+			EqualsFunc: func(old *apps.Deployment, new *apps.Deployment) bool {
+				return reflect.DeepEqual(old.Spec, new.Spec)
+			},
+		}
+
+		waitUntilDeployed()
+	})
+
+	It("[test_id:7369] should set SSPOperatorReconcilingProperly metrics to 0 on failing to reconcile", func() {
+		foundSsp := getSsp()
+		Expect(foundSsp.Status.Phase).To(Equal(lifecycleapi.PhaseDeployed), "SSP should be in phase Deployed")
+		// add a finalizer to the validator deployment, do that it can't be deleted
+		Expect(apiClient.Get(ctx, deploymentRes.GetKey(), deployment)).ToNot(HaveOccurred())
+		controllerutil.AddFinalizer(deployment, finalizerName)
+		//deployment.Finalizers = append(deployment.Finalizers, "kubernetes.io/temp-protection")
```

Comment:
can you please remove this commented code?

Reply:
yes!